### DISCUSSION TITLE
_tw vs TailPress, which one is more suitable for novice developers? I mainly use it in woocommerce

### DISCUSSION THREAD

**Discussion by aboutjquery** [gregsullivan/_tw#145] [General]
I find that _tw is more concise and has no redundant styles, but is there no woocommerce template? I just installed _tw, and the default style of woocommerce is still woocommerce itself. How do I get started? Is there any official documentation? Thank you

**Comment by gregsullivan** [comment]
Hello—I've converted this issue to a discussion as well!

To my knowledge, Tailpress also doesn't have specific WooCommerce support. That said, I've never used Tailpress.

The _tw documentation page that is most applicable to WooCommerce is this one: [Styling HTML from outside the theme](https://underscoretw.com/docs/styling-html-from-outside-the-theme/).

With WooCommerce via _tw, you can either:

* Create new HTML by overriding templates in a `woocommerce` folder in your theme, in which case you could use Tailwind classes in the HTML, or
* [Use the existing HTML](https://underscoretw.com/docs/styling-html-from-outside-the-theme/#h-using-the-existing-output) and override the styles via `@apply` (assuming you want to use Tailwind classes in your CSS files).
